index.html

```diff
@@ -341,38 +313,69 @@ <h1>About me</h1>
         </div>
       </section>
 
-        
-
       <section class="contact-section">
         <div class="contact-heading">
-          <h3>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h3>
+          <h3>
+            I'm always interested in hearing about new projects, so if you'd
+            like to chat please get in touch.
+          </h3>
         </div>
         <div class="form-container">
-          <form action="#" class="form">
-            <input type="text" name="name" id="name" placeholder="Full name">
-            <input type="email" name="email" id="email" placeholder="Email address">
-            <textarea name="textarea" id="textarea" cols="5" rows="5" placeholder="Enter text here"></textarea>
+          <form action="https://formspree.io/f/xrgvbqyj" method="post" class="form">
+            <input type="text" name="name" id="name" placeholder="Full name" maxlength="30" required/>
+            <input
+              type="email"
+              name="email"
+              id="email"
+              placeholder="Email address" required
+            />
+            <textarea
+              name="textarea"
+              id="textarea"
+              cols="5"
+              rows="5"
+              placeholder="Enter text here" maxlength="500" required
+            ></textarea>
```

Comment:
- [ ] I was about to approve your project for the wonderful work you did :clap:, but I noticed that the form should send a message with a successful response. Kindly make sure you are able to receive a message with on your email address , When you click on the `Get in touch` button . :climbing_woman: 

![Screenshot from 2023-01-17 17-38-35 (1)](https://user-images.githubusercontent.com/80458199/212944977-287b3d88-dd87-4c4b-99f2-6fb7e72740af.png)

Reply:
![image](https://user-images.githubusercontent.com/116083031/212951519-eb0bcded-fda5-466a-abaa-072724a29196.png)

Hi! I think you overlooked the code. The changes you requested I have already made the changes you asked for.